src/LocaleService.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+namespace Drupal\argo;
+
+use Drupal\locale\StringDatabaseStorage;
+use Drupal\tableau_i18n_locale\TableauLocaleStringInterface;
```

Comment:
Same consideration as `webform`, should we add the module dependency?

Reply:
I removed the hard dependency. And when we get the @todo below figured out that will be even better